modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -9539,6 +9539,71 @@ public void testPostCustomObjectEntryWithStatus() throws Exception {
 				_objectDefinition1.getRESTContextPath(), Http.Method.POST));
 	}
 
+	@Test
+	public void testPostObjectEntryExpire() throws Exception {
+
+		// Company scoped object entry
+
+		ObjectEntry companyScopedObjectEntry =
+			_objectEntryLocalService.addObjectEntry(
+				0, TestPropsValues.getUserId(),
+				_objectDefinition1.getObjectDefinitionId(),
+				ObjectEntryFolderConstants.
+					PARENT_OBJECT_ENTRY_FOLDER_ID_DEFAULT,
+				null,
+				HashMapBuilder.<String, Serializable>put(
+					_OBJECT_FIELD_NAME_TEXT, RandomTestUtil.randomString()
+				).build(),
+				ServiceContextTestUtil.getServiceContext());
+
+		JSONObject jsonObject = HTTPTestUtil.invokeToJSONObject(
+			null,
+			StringBundler.concat(
+				_objectDefinition1.getRESTContextPath(), "/",
+				companyScopedObjectEntry.getObjectEntryId(), "/expire"),
+			Http.Method.POST);
```

Comment:
I think we have a method in this class that supports that endpoint creation. Please, check `_getEndpoint()`.
Can you use this method and, if you need, you could do something similar to this example:

```
StringBundler.concat(
    _getEndpoint(_objectDefinition1, _testGroupId),
    StringPool.SLASH, id, "/permissions"),
```